### PR TITLE
Explain only the relevant release level.

### PR DESCRIPTION
**Pull request by lukesneeringer**
This commit makes it such that the README only elucidates the release level of the given repo, rather than all of them.

### CODE REVIEW

**Comment by lukesneeringer** [comment]
Since I do not have write on the repo, I can not push a direct branch and there is nothing I can do about the CI failure.

File: src/cli/commands/generate.js [hunk 1]
```diff
@@ -30,6 +30,14 @@ const utils = require('../../utils');
 handlebars.registerHelper('slugify', (str) => string(str).slugify().s);
 handlebars.registerHelper('trim', (str) => string(str).trim().s);
 handlebars.registerHelper('release_quality', utils.createReleaseQualityBadge);
+handlebars.registerHelper('if_eq', (left, right, opts) => {
+  if (left === right) {
+    return opts.fn(this);
+  }
+  else {
```
Comment (jmdobry) [inline src/cli/commands/generate.js]: Remove the `else` block and simply have `return opts.inverse(this);` be the last line of the function.

Comment (lukesneeringer) [reply src/cli/commands/generate.js]: Why? Isn't `if_eq`/`else` useful in a template?

Comment (jmdobry) [reply src/cli/commands/generate.js]: Yes, keep the helper function. I'm just saying change the code to:

```js
handlebars.registerHelper('if_eq', (left, right, opts) => {
  if (left === right) {
    return opts.fn(this);
  }
  return opts.inverse(this);
});
```

Comment (lukesneeringer) [reply src/cli/commands/generate.js]: Oh I see. Okay.

**Pull request merged**